src/MOI/MOI_wrapper.jl

```diff
@@ -566,9 +566,8 @@ function reset_message_callback(model)
         removecbmessage(model.inner, C_NULL, C_NULL)
         model.message_callback = nothing
     end
-    if Sys.iswindows() &&                   # is win
-            model.inner.logfile == "" &&    # no file -> screen
-            model.log_level != 0            # has log
+    if model.inner.logfile == "" &&    # no file -> screen
```

Comment:
Linux also requires this?

Reply:
According to FICO:

> Prior to Xpress 8.9, the Optimizer library would automatically print messages to stdout for all Unix-like platforms
> (including MacOS) if no message callback function was defined. With Xpress 8.9 and later, the Optimizer library will no longer print messages automatically for any platform.
> 
> To print the messages produced by the Optimizer, a user should set a message callback, using XPRSaddcbmessage():
> 
> https://www.fico.com/fico-xpress-optimization/docs/latest/solver/optimizer/HTML/XPRSaddcbmessage.html
> 
> and provide it with a function that prints each message to stdout, as in the example provided in the link above.

Reply:
What if you add:
```yaml
os: [ubuntu-latest, windows-latest, macOS-latest]
```
to the CI? Then all major platforms will be tested automatically.a

Reply:
> Linux also requires this?


Short answer: yes.